staging_service/import_specifications/individual_parsers.py

```diff
@@ -101,22 +101,25 @@ def _check_for_duplicate_headers(
         seen.add(name)
 
 
+# This method expects to be called from the _parse_xsv method, and therefore
```

Comment:
Maybe the comment goes into the method itself?

Reply:
Doesn't that mean it winds up as a docstring? I put it outside the method to avoid that

Reply:
Nope

```
In [1]: def foo(): 
   ...:     """This is a docstring""" 
   ...:                                                                         

In [2]: def bar(): 
   ...:     # This is a docstring 
   ...:     pass 
   ...:                                                                         

In [3]: foo.__doc__                                                             
Out[3]: 'This is a docstring'

In [4]: bar.__doc__                                                             

```

Reply:
Fixed